template_feed/Microsoft.DotNet.Common.ProjectTemplates.3.0/content/ConsoleApplication-FSharp/Program.fs

```diff
@@ -2,7 +2,13 @@
 
 open System
 
+// Define a function to print a message
+// Functions are defined before they are used in F#
+let printMessage message =
+    printfn "%s" message
+
 [<EntryPoint>]
 let main argv =
-    printfn "Hello World from F#!"
+    // Call the previously-defined function
+    printMessage "Hello world from F#!"
```

Comment:
I think it's really a good idea @cartermp 

just a feedback, maybe 

```fsharp
let hello from =
     sprintf "Hello world from %s!" from

let main argv =
    let message =  hello "F#"	    // Call the previously-defined function
    printfn "%s" message
```

can do the same (show usage of functions BEFORE main), but:
- show how to create a binding
- show `sprintf` usage to create strings

maybe is just me, but `printMessage` and `printfn` seems confusing

Reply:
That would be great, I'll do something like that 👍